views/js/qtiCreator/editor/gridEditor/resizable.js

```diff
@@ -23,209 +23,204 @@ define([
     'taoQtiItem/qtiCreator/editor/gridEditor/helper',
     'taoQtiItem/qtiCreator/helper/qtiElements',
     'taoQtiItem/lib/jqueryui_dragdrop'
-], function(_, $, config, helper, qtiElements){
-    "use strict";
-    let _syncHandleHeight = function($row){
+], function (_, $, config, helper, qtiElements) {
+    'use strict';
+
+    // in one row max 12 columns
+    const maxUnits = 12;
+
+    let _syncHandleHeight = function ($row) {
         let h = $row.height() - parseFloat($row.children('[class^="col-"], [class*=" col-"]').css('margin-bottom'));
         $row.find('.grid-edit-resizable-zone').height(h);
     };
 
-    const _createResizables = function createResizables($el){
-
-           let marginWidth = parseFloat($el.find('[class^="col-"]:last, [class*=" col-"]:last').css('margin-left'));
-           let activeWidth = 20;
-
-
+    const _deleteResizables = function _deleteResizables($el) {
+        $el.find('.grid-edit-resizable-zone').remove();
+    };
 
-        $el.find('[class^="col-"], [class*=" col-"]').each(function(){
+    let _setColUnits = function _setColUnits($elt, newUnits) {
+        if ($elt.attr('class').match(/col-([\d]+)/)) {
+            let oldUnits = $elt.attr('data-units');
+            let $parentRow = $elt.parent('.grid-row') || maxUnits;
```

Comment:
$parentRow name with $ sign indicates its should be jqeury element where for sure maxUnits is integer

Reply:
indeed, if is expecting a jquery object, the below `$parentRow.attr(...)` can fail right?